content/en/_index.html

```diff
@@ -13,30 +13,322 @@
 #    icon: watch.svg
 ---
 
-{{< blocks/cover image_anchor="top" height="min" color="primary" >}}
-<img src="notary-horizontal-white.png" class="site-logo" />
-<h2 class="mt-4">
-  The Notary project comprises a server and a client for running and interacting
-with trusted collections.
-</h2>
+{{< blocks/cover image_anchor="top" height="max" color="black" >}}
+<section
+  class="flex gap-lg width-80 fullwidth-mobile is-centered is-align-items-center margin-top-mobile is-mobile-responsive-reverse fullwidth-tablet"
+>
+  <div class="text-left text-centered-mobile">
+    <h4 class="title">
+      Standards-based spec and tooling
+      <span class="text-blue-gradient"
+        >for securing software supply chains</span
+      >
+    </h4>
+    <p class="subtitle">
+      Signing and verifying artifacts. Safeguarding the software delivery
+      security from development to deployment.
+    </p>
+    <div class="flex gap-md margin-lg-top text-centered-mobile">
+      <a href="https://notaryproject.dev/docs/quickstart/">
+        <button class="button is-medium is-dark get-started">
+          Get started
+        </button>
+      </a>
+      <a href="https://github.com/notaryproject/notation">
+        <button class="button is-medium is-dark demo">GitHub</button>
+      </a>
+    </div>
+  </div>
+  <div class="img-long is-hidden-mobile">
+    <iframe
+      src="https://www.youtube.com/embed/TU7a0xUcBjk"
+      frameborder="0"
+    ></iframe>
+  </div>
+</section>
+<section>
+  <p><b>Contributed by the community, in collaboration with</b></p>
+  <div
+    class="width-80 is-centered flex space-between is-align-items-center is-mobile-responsive text-centered-mobile"
+  >
+    <div class="img-container-smd is-centered">
+      <img src="aws_logo.svg" alt="AWS logo" />
+    </div>
+    <div class="img-container-mdl is-centered">
+      <img src="cncf_logo.svg" alt="CNCF logo" />
```

Comment:
I am not sure how "In collaboration with CNCF" sounds in this context. Notary is a CNCF project but I think we should list only collaborators.

Reply:
This list includes major contributor and collaborator organizations. It's fine to add CNCF's logo to the contributor org list since CNCF has employees who are contributing to Notary